scripts/ms

```diff
@@ -35,6 +35,7 @@ def parse(args):
     parser.add_argument('config')
     parser.add_argument('-n', '--n-processes', default=1, type=int)
     parser.add_argument('-v', '--verbose', action='store_true')
+    parser.add_argument('-a', '--annual', action='store_true')
```

Comment:
We talked about it when the time chunking was done through `xarray` so the options were limited by the `groupBy` function.  I think how we do it now this would be relatively easy.  I'll add an issue for this and can address it in another pull request since this one's already pretty big and complicated.